api/sql/schema/360$rule.limit.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE [rule].[limit] (
 	maxCountWeekly BIGINT,
 	maxAmountMonthly MONEY,
 	maxCountMonthly BIGINT,
+    isMaskChecked BIT NOT NULL DEFAULT(0),
```

Comment:
Rename this field to "credentials" and change type to bigint. We will use it as default mask to check against. We should allow it to be NULL, which means credentials will not be checked by default.

Also add column for priority (smallint). This will allow same condition to have limits with different credentials needed (including no credentials). There should be unique constraint for conditionId + priority.